### ISSUE TITLE
Show indexing loading status

### ISSUE THREAD

**Issue by jasonLaster** [RecordReplay/devtools#1564]
**Description:**
At the moment we show a skeleton as we prepare the "basic" loading data needed to view the video and some other basic data....

We should also indicate that we're loading the fully execution ready version of the recording, things needed for breakpoints and other things...

- analysis
- frame steps
- pauses...

**Comment by jasonLaster** [comment]
TODO: get list of actions which are blocked

**Comment by jasonLaster** [comment]
ideas
- when i hover in the gutter, i should see an "indexing..."  tooltip
- show a global loader for indexing statu

**Comment by bhackett1024** [comment]
I think there should be a global indication of how loading is doing, since it can take quite a while if the recording is long and we were able to load the recording's basic information from persistent storage.

**Comment by jonbell-lot23** [comment]
Hey @jaril ! Jason and I were talking and we thought this might be a fun design task for you to take on, to get out of the code a bit :)

**Comment by jonbell-lot23** [comment]
I remember we changed our thinking here from "one global loader" to "show things per element."

Should this bug be closed? I think we did the per element thing.

**Comment by jasonLaster** [comment]
We've definitely flip flopped here, but we haven't implemented something
yet and I do think we want to show folks that we're still indexing.

My simple preference would be to show a global loading spinner.

re: individual elements, i think it's really hard at the moment to come up
with all the places to do that.

**Comment by jonbell-lot23** [comment]
Scopes, callstack, evaluating in the console, elements panel, and breakpoints are all designed, discussed, put in Figma, and documented in Notion here -> https://www.notion.so/replayio/Loading-UX-89ec80c7d09e461e8500516638deec33 

We spent a lot of design time trying to do a global progress bar but the design POV was that we show loading per element. There are a few reasons for this, but the most noticeable is how we'd be showing two progress bars on every load, one on top and one on bottom.

I propose our next steps are making production line up with where we landed in Notion.
